ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ExceptionRequestsPage.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import {
+    Flex,
+    FlexItem,
+    PageSection,
+    Tab,
+    TabTitleText,
+    Tabs,
+    Title,
+} from '@patternfly/react-core';
+
+type TabKey = 'PENDING_APPROVALS' | 'APPROVED_DEFERRALS' | 'APPROVED_FALSE_POSITIVES';
+
+function ExceptionRequestsPage() {
+    const [activeTabKey, setActiveTabKey] = useState<TabKey>('PENDING_APPROVALS');
+
+    const handleTabClick = (event, tabIndex) => {
+        setActiveTabKey(tabIndex);
+    };
+
+    return (
+        <>
```

Comment:
Small nit: can we add a `<PageTitle>` component here as well?

Reply:
I decided not to add it here because we'll be adding different sections (ie. Pending approvals, etc.) so I was thinking of adding it to those sections